roles/variables/vars/main.yml

```diff
@@ -101,11 +101,11 @@ zookeeper_combined_properties: "{{ zookeeper_properties | confluent.platform.com
 zookeeper_final_properties: "{{ zookeeper_combined_properties | combine(zookeeper_custom_properties) }}"
 
 #### Kafka controller variables
-kafka_controller_service_name: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka'}}"
+kafka_controller_service_name: "confluent-kcontroller"
```

Comment:
why are we not keeping a separate name for community kafka?

Reply:
for the broker, we are still doing the same separation meaning different names on the basis of whether it's a community kafka or confluent Kafka. I don't see a reason to have an other name for the controller though.
let's discuss more on this in our next tech discussion session.